<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-7SBKKJ1" failures="3" tests="12" name="PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report" time="115.410" errors="0" timestamp="2024-12-05T14:26:19 IST" skipped="0">
  <testcase classname="PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report" name="Username" time="1.458"/>
  <system-out/>
  <testcase classname="PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report" name="Password" time="1.509"/>
  <system-out/>
  <testcase classname="PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report" name="Captcha" time="2.320"/>
  <system-out/>
  <testcase classname="PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report" name="SUBMIT_Button" time="2.217"/>
  <system-out/>
  <testcase classname="PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report" name="Click_On_PSF" time="20.608"/>
  <system-out/>
  <testcase classname="PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report" name="Remove_PouP" time="0.680"/>
  <system-out/>
  <testcase classname="PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report" name="Click_On_Static_Reports" time="1.097"/>
  <system-out/>
  <testcase classname="PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report" name="Click_On_Preliminary_NPS_Report" time="5.120"/>
  <system-out/>
  <testcase classname="PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report" name="Click_On_Workshop" time="30.652">
    <failure message="No Data Found for Workshop_Wise In Premilinary_NPS_Report : 0%" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: No Data Found for Workshop_Wise In Premilinary_NPS_Report : 0%
at org.testng.Assert.fail(Assert.java:99)
at PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report.Click_On_Workshop(Prelimnary_NPS_report.java:155)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Click_On_Workshop -->
  <system-out/>
  <testcase classname="PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report" name="Click_On_Day_Wise" time="24.940">
    <failure message="No Data Found for Day_Wise In Premilinary_NPS_Report : 0 %" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: No Data Found for Day_Wise In Premilinary_NPS_Report : 0 %
at org.testng.Assert.fail(Assert.java:99)
at PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report.Click_On_Day_Wise(Prelimnary_NPS_report.java:214)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Click_On_Day_Wise -->
  <system-out/>
  <testcase classname="PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report" name="Click_On_Service_Type_Wise" time="24.809">
    <failure message="No Data Found for Service_Type_Wise In Premilinary_NPS_Report : 0 %" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: No Data Found for Service_Type_Wise In Premilinary_NPS_Report : 0 %
at org.testng.Assert.fail(Assert.java:99)
at PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report.Click_On_Service_Type_Wise(Prelimnary_NPS_report.java:273)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Click_On_Service_Type_Wise -->
  <system-out/>
  <testcase classname="PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report" name="Comparison_0f_all_Data" time="0.000"/>
  <system-out/>
</testsuite> <!-- PSf_Static_Report_and_Call_Dispostion.Prelimnary_NPS_report -->
